### PR TITLE
Add a deprecation warning

### LINKED ISSUE #2412: Show a banner describing the future of Appmaker
We should have a banner on Appmaker which describes what's happening to the tool and the makes (probably just a link to https://blog.webmaker.org/whats-next-for-webmaker-tools).

Something like...
```
<div style="z-index: 9999; position: absolute; top: 0; left: 0; right: 0; height: 2em; background-color: red; color: white; text-align: center; padding-top: 0.25em;">
  <div style="position: absolute; right: 0; height: 100%; width: 50px; cursor: pointer;" onclick="this.parentNode.parentNode.removeChild(this.parentNode)">X</div>
  Hey! Webmaker is changing, and Appmaker will be put on ice indefinitely.
  <a href="https://blog.webmaker.org/whats-next-for-webmaker-tools" style="color: white;">Check it.</a>
</div>
```

screenshot 2015-06-03 16 41 12

Blocks #2409

### PR DESCRIPTION
**Pull request by errietta**
Closes #2412

RIP appmaker :sob:

### CODE REVIEW

**Comment by secretrobotron** [comment]
Thanks @errietta. Check those tests. Travis is failing.

File: views/designerHTML.ejs [hunk 1]
```diff
@@ -1,8 +1,14 @@
+<div class="deprecation" id="deprecation">
+  <div class="closeWarning" id="closeWarning" onclick="this.parentNode.remove()">X</div>
+    Mozilla's tools for teaching the Web are evolving. Beginning June 19, you will not be able to create or edit Appmaker projects. To learn more, read <a href="http://mzl.la/1ACom34" target="_blank">our blog post.</a>
```
Comment (secretrobotron) [inline views/designerHTML.ejs]: This needs to be transformed by i18n. Check the rest of the `ejs` code for some examples about how to do that.

File: public/stylesheets/style.less [hunk 2]
```diff
@@ -2012,7 +2012,7 @@ button.return-btn {
 // Basic Page Layout
 
 .page-wrapper {
-  top: 0px;
```
Comment (secretrobotron) [inline public/stylesheets/style.less]: Maybe put all of the `top` edits you made into some classes. Then you can easily turn the banner on and off.

**Comment by errietta** [comment]
@secretrobotron R

**Comment by secretrobotron** [comment]
@errietta thanks for this! Would love that text to fit a bit more like the popcorn maker banner does. See how it squishes into the X here?
screenshot 2015-07-24 18 25 31

**Comment by errietta** [comment]
@secretrobotron is it better?

**Comment by Pomax** [comment]
A nit: "moving forward" is corporate-speak, padding a sentence without adding any meaning (we don't get a choice in which direction we experience time, so *everything* happens "moving forward"). Since we want to engage people at a normal language level, rather than corporate-speak, I'd recommend omitting those two words. They just makes us sound needlessly posh.

**Comment by secretrobotron** [comment]
@errietta ok, got some details about the right string to put in there for you:

> Our tools for teaching the Web are evolving. As of September 30th, Mozilla will no longer provide support for Appmaker. To learn more, read our blog post [http://mzl.la/webmaker-tools-transition].

Pull request incoming.

cc @kevzawacki

**Comment by KevZawacki** [comment]
Also (pardon the last-minute copyedit), can we change "Mozilla will no longer provide support for Appmaker" to "Mozilla will no longer support Appmaker."

**Comment by secretrobotron** [comment]
:+1:

**Comment by davidascher** [comment]
Can we make the link not blue on red please? Low color contrast hurts
legibility.

**Comment by secretrobotron** [comment]
screenshot 2015-07-27 18 05 21

@vazquez @flukeout sane colours? Could be better?

**Comment by secretrobotron** [comment]
@errietta a pull request for you on your branch :) : https://github.com/errietta/appmaker/pull/1

**Comment by errietta** [comment]
@secretrobotron merged your code in my branch

**Comment by vazquez** [comment]
@secretrobotron That colour is totally fine :rainbow: 

I would `text-align: left` and make the close icon bigger

**Comment by secretrobotron** [comment]
:+1: will do

**Comment by secretrobotron** [comment]
@errietta another pull request for you on @vazquez's recommendations

**Comment by errietta** [comment]
merged in @secretrobotron  :cat:

File: views/designerHTML.ejs [hunk 3]
```diff
@@ -1,3 +1,17 @@
+<div class="deprecation" id="deprecation">
+  <a class="closeWarning" id="closeWarning">&times;</a>
+  Our tools for teaching the Web are evolving. As of September 30th, Mozilla will no longer support Appmaker. To learn more, read  <a href="http://mzl.la/webmaker-tools-transition" target="_blank">our blog post.</a>
+  <script>
+    (function () {
+      var closeWarning = document.querySelector('#closeWarning');
+      closeWarning.onclick = function (e) {
+        closeWarning.parentNode.remove();
```
Comment (Pomax) [inline views/designerHTML.ejs]: technically, `remove()` is not a function in the JS DOM API (it's available in some consoles, which shim some jQuery-esque functions in). If vanilla JS, to remove the closeWarning, this should be `closeWarning.parentNode.removeChild(closeWarning)`. If the intention is to remove the closeWarning's parent, let's give that the `id` instead, and then remove it directly.

Comment (secretrobotron) [inline views/designerHTML.ejs]: Good call. Will fix.

**Comment by secretrobotron** [comment]
@errietta one more for you :).

**Comment by secretrobotron** [comment]
@ScottDowne says R+ after that extra patch is merged.

**Pull request merged**